tests/manage/pv_services/test_raw_block_pv.py

```diff
@@ -0,0 +1,165 @@
+            access_mode=constants.ACCESS_MODE_RWX,
+            namespace=self.namespace,
+            volume_mode='Block'
+        )
+
+        pvcs = [pvc_mb, pvc_gb, pvc_tb]
+
+        for pvc in pvcs:
+            helpers.wait_for_resource_state(
+                resource=pvc, state=constants.STATUS_BOUND
+            )
+
+        pvs = [pvc.backed_pv_obj for pvc in pvcs]
+
+        pod_dict = constants.CSI_RBD_RAW_BLOCK_POD_YAML
+        pvc_mb_pods = [(helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_mb.name,
+            namespace=self.namespace,
+            raw_block_pv=True,
+            pod_dict_path=pod_dict,
+            node_name=random.choice(
+                worker_nodes)
+        )
+        ) for _ in range(3)
+        ]
+
+        pvc_gb_pods = [(helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_gb.name,
+            namespace=self.namespace,
+            raw_block_pv=True,
+            pod_dict_path=pod_dict,
+            node_name=random.choice(
+                worker_nodes)
+        )
+        ) for _ in range(3)
+        ]
+
+        pvc_tb_pods = [(helpers.create_pod(
+            interface_type=constants.CEPHBLOCKPOOL,
+            pvc_name=pvc_tb.name,
+            namespace=self.namespace,
+            raw_block_pv=True,
+            pod_dict_path=pod_dict,
+            node_name=random.choice(
+                worker_nodes
+            )
+        )
+        ) for _ in range(3)
+        ]
+
+        def flatten(l):
+            return [item for sublist in l for item in sublist]
+        pods = [pvc_mb_pods, pvc_gb_pods, pvc_tb_pods]
+        pods = flatten(pods)
+        for pod in pods:
+            helpers.wait_for_resource_state(
+                resource=pod, state=constants.STATUS_RUNNING, timeout=120)
+        storage_type = 'block'
+
+        with ThreadPoolExecutor() as p:
+            for pod in pvc_mb_pods:
+                logging.info(f'running io on pod {pod.name}')
+                p.submit(
+                    pod.run_io, storage_type=storage_type, size=f'{random.randint(10,200)}M',
+                )
+            for pod in pvc_gb_pods:
+                logging.info(f'running io on pod {pod.name}')
+                p.submit(
+                    pod.run_io, storage_type=storage_type, size=f'{random.randint(1,5)}G',
+                )
+            for pod in pvc_tb_pods:
+                logging.info(f'running io on pod {pod.name}')
+                p.submit(
+                    pod.run_io, storage_type=storage_type, size=f'{random.randint(10,15)}G',
+                )
+
+        for pod in pods:
+            get_fio_rw_iops(pod)
+        return pods, pvcs, pvs
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-750")
+@pytest.mark.usefixtures(
+    create_rbd_secret.__name__,
+    create_ceph_block_pool.__name__,
+    create_rbd_storageclass.__name__,
+)
+class TestRawBlockPVRetain(BaseRawBlockPV):
```

Comment:
pretty sure you can combine all test classes under one class and use pytest parametrize